blockly-rtc/src/WorkspaceClient.js

```diff
@@ -48,15 +49,22 @@ export default class WorkspaceClient {
 
 
   /**
-   * Initiate the workspace by running all stored events and activating the
-   * handling of recieving events from the server.
+   * Initiate the workspace by loading the current workspace and activating the
+   * handlers for sending and recieving events between the client and server.
    * @public
    */
-  async initiateWorkspace() {
-    // TODO: Get an SVG of the current blocks on the workspace instead of
-    // replaying history.
-    const events = await this.getEventsHandler(0);
+  async start() {
+    // Load the current state of the workspace.
+    const workspace = Blockly.Workspace.getById(this.workspaceId);
+    const snapshot = await this.getSnapshotHandler();
```

Comment:
Will snapshot ever be null?

Reply:
It should not be, the server has a valid default value if it has not been set. I can only think of the case where there are errors retrieving it from the server in which case this.getSnapshotHandler() will reject. Is this a case I should account for?

Reply:
Nope, that sounds fine.